packages/test-interface-criteria/src/criteria/criteriaUtils.js

```diff
@@ -77,15 +81,27 @@ function mergeDateRangeNodeConstraints(reqA, reqB) {
 }
 
 /**
- * @param {NumericNodeConstraint} reqA
- * @param {NumericNodeConstraint} reqB
+ * @param {NumericNodeConstraint} baseConstraint
+ * @param {NumericNodeConstraint} extensionConstraint
  * @returns {NumericNodeConstraint}
  */
-function mergeNumericNodeConstraints(reqA, reqB) {
+function mergeNumericNodeConstraints(baseConstraint, extensionConstraint) {
   return {
     '@type': 'NumericNodeConstraint',
-    ...mergeTestDataNodeConstraintField('mininclusive', reqA, reqB, Math.max),
-    ...mergeTestDataNodeConstraintField('maxinclusive', reqA, reqB, Math.min),
+    ...mergeTestDataNodeConstraintField(
+      'mininclusive',
+      baseConstraint,
+      extensionConstraint,
+      // The extension should overwrite the base if they both exist
+      (base, extension) => extension,
+    ),
+    ...mergeTestDataNodeConstraintField(
+      'maxinclusive',
+      baseConstraint,
+      extensionConstraint,
+      // The extension should overwrite the base if they both exist
+      (base, extension) => extension,
```

Comment:
This makes more sense to me given that the language is of a Criteria "extending" another. This is a simpler and more explicit means of doing that